manifests/0000_30_cluster-api_15_allow-egress-controllers.yaml

```diff
@@ -0,0 +1,125 @@
+# This NetworkPolicy allows specific egress traffic required for cluster-api controllers
```

Comment:
Are we supposed to be going so granular on egress? I had assumed we would allow all egress from our pods as they need to be able to communicate with the internet for various API requests

Some of the egress rules here (e.g. the metrics ports) I wouldn't have expected to see in the egress. Our various controllers wouldn't be calling the metrics/webhooks of other pods within the namespace would they? I'm assuming this is destination port that we are filtering on

Reply:
yes reviewing this more .